tsconfig.json

```diff
@@ -1,17 +1,31 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": ["es6", "dom"],
     "declaration": true,
     "declarationMap": true,
-    "sourceMap": true,
-    "outDir": "dist",
     "esModuleInterop": true,
     "exactOptionalPropertyTypes": true,
-    "strict": true,
     "forceConsistentCasingInFileNames": true,
+    "lib": ["es6", "dom"],
     "module": "commonjs",
-    "moduleResolution": "node"
+    "moduleResolution": "node",
+    "outDir": "dist",
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "strict": true,
+    "target": "es5",
+    // Override types from @polkadot/api-augment with generated types from Tangle
+    // Ref: https://polkadot.js.org/docs/api/examples/promise/typegen/#typescript-config
+    "baseUrl": "node_modules",
```

Comment:
The "baseUrl" here shouldn't be set to node_modules. Instead, it should be relative to the root of the project.